### PR TITLE
feat: disable global rule for checking TODO comments

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**

### CODE REVIEW

**Comment by xiaozhenliu-gg5** [comment]
ref: 1) https://github.com/googleapis/gapic-generator-typescript/pull/343
       2) https://github.com/googleapis/gapic-generator-typescript/pull/340

File: .eslintrc.json [hunk 1]
```diff
@@ -27,11 +27,16 @@
...
         "@typescript-eslint/no-use-before-define": "off",
+        "@typescript-eslint/no-warning-comments": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/camelcase": "off",
+        "@typescript-eslint/no-var-requires": "off",
         "node/no-missing-import": "off",
+        "node/no-empty-function": "off",
         "node/no-unsupported-features/es-syntax": "off",
         "node/no-missing-require": "off",
+        "node/no-unused-vars": "off",
```
Comment (JustinBeckwith) [inline .eslintrc.json]: Lets drop `no-unused-vars`

File: .eslintrc.json [hunk 2]
```diff
@@ -27,11 +27,16 @@
       "rules": {
         "@typescript-eslint/no-non-null-assertion": "off",
         "@typescript-eslint/no-use-before-define": "off",
+        "@typescript-eslint/no-warning-comments": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/camelcase": "off",
+        "@typescript-eslint/no-var-requires": "off",
```
Comment (JustinBeckwith) [inline .eslintrc.json]: Lets drop `no-var-requires`

File: .eslintrc.json [hunk 3]
```diff
@@ -27,11 +27,16 @@
...
         "node/no-missing-import": "off",
+        "node/no-empty-function": "off",
         "node/no-unsupported-features/es-syntax": "off",
         "node/no-missing-require": "off",
+        "node/no-unused-vars": "off",
+        "node/no-explicit-any": "off",
```
[the remark this answers is not in the document]
Comment (xiaozhenliu-gg5) [reply .eslintrc.json]: done.

File: .eslintrc.json (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply .eslintrc.json]: done.

File: .eslintrc.json (hunk 2 above)
Comment (xiaozhenliu-gg5) [reply .eslintrc.json]: done.

**Pull request merged**